Rubberduck.Parsing/Symbols/IdentifierReferenceResolver.cs

```diff
@@ -84,11 +84,7 @@ public void EnterWithBlock(VBAParser.WithStmtContext context)
 
         private IBoundExpression GetInnerMostWithExpression()
         {
-            if (_withBlockExpressions.Any())
-            {
-                return _withBlockExpressions.Peek();
-            }
-            return null;
+            return _withBlockExpressions.Any() ? _withBlockExpressions.Peek() : null;
```

Comment:
I feel like there should be a `PeekOrDefault`.

Reply:
I'm not yet familiar with `PeekOrDefault` I didn't see that method in https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.stack-1?view=netframework-4.7.1.